CMakeLists.txt

```diff
@@ -41,3 +41,4 @@ unset(BUILD_UNIT_TESTS CACHE)
 unset(BUILD_TEST_MODULES CACHE)
 unset(BUILD_EXAMPLE_MODULES CACHE)
 unset(USE_TLS CACHE)
+unset(FORCE_DEFRAG CACHE)
```

Comment:
We should add this to some CI presumably. The daily ASAN probably?

Reply:
Yes I agree. maybe it can also be run as part of the CI, depending on the extra time it would take.